packages/massa-web3/src/experimental/basicElements/serializers/bignum.ts

```diff
@@ -0,0 +1,158 @@
+import { U64_MAX } from './numbers'
```

Comment:
What about:
- creating u64, u128, u256 types as wrapper of bigint?
- adding a from and to generic function that convert everything (maybe at the index level)?